doc/connectivity/bluetooth/api/mesh/rpr_cli.rst

```diff
@@ -0,0 +1,25 @@
+.. _bluetooth_mesh_models_rpr_cli:
+
+Remote Provisioning Client
+##########################
+
+The Remote Provisioning Client model is a foundation model defined by the Bluetooth
+mesh specification. It is enabled with the
+:kconfig:option:`CONFIG_BT_MESH_RPR_CLI` option.
+
+The Remote Provisioning Client model is introduced in the Bluetooth Mesh Protocol
+Specification version 1.1, and provides functionality to remotely provision devices into a mesh
+network by interacting with mesh nodes that supports the :ref:`bluetooth_mesh_models_rpr_srv` model.
+
+The Remote Provisioning Client model communicates with a Remote Provisioning Server model
+using the device key of the node containing the target Remote Provisioning Server model instance.
+
+If present, the Remote Provisioning Client model must be instantiated on the primary
+element. It may also be supported by any secondary element.
```

Comment:
We don't really support this as per my understanding. There is no check in `rpr_cli_init` that another RPR client instance was instantiate already, but even if we could instantiated several RPR clients, we can't provision multiple devices in parallel (we keep only one provisioning context). So I'd suggest to remove the second sentence and add check to `rpr_cli_init` in `rpr_cli.c` that element index is zero (like in server).